corehq/apps/analytics/tasks.py

```diff
@@ -369,3 +365,13 @@ def _track_periodic_data_on_kiss(submit_json):
         s3_connection.upload(filename, f, 'kiss-uploads')
 
     os.remove(filename)
+
+
+def _log_response(data, response):
+    try:
+        response_text = json.dumps(response.json(), indent=2, sort_keys=True)
+    except:
```

Comment:
`except Exception` is probably preferred here since this will also swallow keyboard interrupts and such